docs/guides/hardhat-walkthrough/hardhat-base-setup.md

```diff
@@ -0,0 +1,78 @@
+---
+sidebar_label: Hardhat basic setup
+sidebar_position: 1
+---
+
+# Setup your Hardhat project
+
+In this article will guide you through the process of seting up an Hardhat installation (using TypeScript), adding the [`@lukso/lsp-smart-contracts`](https://www.npmjs.com/package/@lukso/lsp-smart-contracts) package, creating a basic LSP7 contract, and deploying it on [LUKSO Testnet](http://docs.lukso.tech/networks/testnet/parameters).
```

Comment:
I suggest you to add which version you used for this guide

Reply:
done